common/templates/context_funcs.go

```diff
@@ -1632,6 +1632,73 @@ func (c *Context) tmplEditChannelTopic(channel interface{}, newTopic string) (st
 	return "", err
 }
 
+const ChannelTagLimit = 5
```

Comment:
Is this a Discord limitation, or arbitrarily set?
If it is a Discord limitation, consider returning the error thrown by
Discord over throwing our own and obfuscating the source.

Reply:
Was a Discord limit I hardcoded into it. Fixed in latest commit